wps-cloudformation-template.yaml

```diff
@@ -57,7 +57,44 @@ Parameters:
   geoserverWpsEndpoint:
     Type: String
     Default: 'http://geoserver-wps.aodn.org.au/geoserver/wps'
+  environment:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - production
+      - systest
+      - sandbox
+      - dev
+  createProdSystestBucket:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+Mappings:
+  EnvironmentMap:
+    production:
+      BucketName: "aws-wps-production"
+    systest:
+      BucketName: "aws-wps-systest"
+Conditions:
+  CreateProdSystestBucket: !And
+    - !Equals [!Ref createProdSystestBucket, true]
+    - !Not [!Condition CreateSandboxDevBucket]
+  CreateSandboxDevBucket: !Not [!Or [!Equals [ !Ref environment, production ], !Equals [ !Ref environment, systest ]]]
```

Comment:
If using the 'default' as fallback, this could just check whether it is equal to default, rather than checking for everything that it *isn't*.

Reply:
That will be true if we have 3 allowed values i.e prod, systest and dev. I am using 4 and therefore extra logic.

Reply:
I think this could just be replaced with:
```yaml
[!Equals [ !Ref environment, default ]
```
If it isn't "default", it should get picked up by the other condition.